docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -83,15 +83,15 @@ When accessing the executor Pod's logs the following error is seen.
 
 *What it means:*
 
-A `NoRouteToHostException` indicates that this Pod is not able to connect to the {WebName} Pod.
+A `NoRouteToHostException` indicates that this pod is not able to connect to the `{WebName}` pod.
 
 *How to resolve it:*
 
-In a new deployment this is expected, as the executor starts before the {WebName}. Otherwise, examine the {WebName} Pod and resolve any errors seen here. Once the {WebName} is running successfully this error should be resolved.
+In a new deployment this is expected, as the executor starts before the {WebName}. Otherwise, examine the `{WebName}` pod and resolve any errors seen here. Once the {WebName} is running successfully this error should be resolved.
 
-.Pod reports insufficient resources
+.A pod reports insufficient resources
 
-After attempting to deploy {WebNameTitle} on OpenShift, the Pod is unable to start, and the following error is seen in the `Events` tab.
+After attempting to deploy {WebNameTitle} on OpenShift, the pod is unable to start, and the following error is seen in the `Events` tab:
```

Comment:
Events tab needs a context. Where is this tab? Check with Marco. Ganesh might also be able to help if he has managed to install from a template on OCP 3.11 or 4.x.